corehq/motech/generic_inbound/views.py

```diff
@@ -157,7 +157,7 @@ def post(self, request, domain, **kwargs):
 @json_error
 @api_auth
 @api_throttle
-@require_http_methods(["POST"])
+@require_http_methods(list(RequestLog.RequestMethod))
```

Comment:
note the commit message comment - this change makes the view allow POST, PUT, and PATCH requests.  I don't think we intend to treat those differently any time soon, and this seemed a reasonable change.